examples/basics/visuals/colorbar_visual.py

```diff
@@ -81,7 +83,7 @@ def get_mandlebrot_escape_values(width, height):
     return v_get_num_escape_turns(*grid).astype(np.float)
 
 
-def get_vertical_bar():
+def get_vertical_bar(pos, halfdim):
```

Comment:
Yeah. I like the "major" / "minor" terminology, I'd use that in the `docstring` instead of any mention of width/height.